errors/apiErrors.ts

```diff
@@ -130,7 +130,10 @@ export function getAxiosErrorWithContext(
   error: AxiosError<any>,
   context: AxiosErrorContext = {}
 ): Error {
-  const { status } = error;
+  let status;
```

Comment:
You could shorten this to `const status = error.response?.status`. Only mentioning it because it's slightly more consistent with what we're already doing in other spots in this file ([example](https://github.com/HubSpot/hubspot-local-dev-lib/pull/127/files#diff-9d5b4d724ee9d55d1fa4e316c00ad65c422dc4365c603388afd8f012d3327158R34))